pkg/cmd/container/create.go

```diff
@@ -817,6 +818,29 @@ func generateLogConfig(dataStore string, id string, logDriver string, logOpt []s
 	return logConfig, nil
 }
 
+func generateRemoveStateDirFunc(ctx context.Context, id string, internalLabels internalLabels) func() {
+	return func() {
+		if rmErr := os.RemoveAll(internalLabels.stateDir); rmErr != nil {
+			log.G(ctx).WithError(rmErr).Warnf("failed to remove container %q state dir %q", id, internalLabels.stateDir)
+		}
+	}
+}
+
+func generateRemoveOrphanedDirsFunc(ctx context.Context, id, dataStore string, internalLabels internalLabels) func() {
```

Comment:
Since it deletes one or more of the following directories, the function name is Dirs.

- `/var/lib/nerdctl/1935db59/containers/<namespace>/<container id>`
- `/var/lib/nerdctl/1935db59/etchosts/<namespace>/<container id>`